py/circuitpy_mpconfig.mk

```
@@ -109,6 +109,12 @@ CIRCUITPY_DIGITALIO = 1
 endif
 CFLAGS += -DCIRCUITPY_DIGITALIO=$(CIRCUITPY_DIGITALIO)
 
+ifndef CIRCUITPY_COUNTIO
+CIRCUITPY_COUNTIO = 1
+endif
```

I expected you to delete the `ifndef` and `endif` and leave the `?=` line.
```suggestion
CIRCUITPY_COUNTIO ?= 1
```